### ISSUE TITLE
Compact layout mode for power-users (less vertical whitespace)

### ISSUE THREAD

**Issue by ara4n** [vector-im/riot-web#109]

**Comment by discordianfish** [comment]
Why power users? Is desktop usage now synonymous with power user? :)

But +1000, I love vector.im but the UI.. I find it really hard to follow the chat passively while working.

Maybe it's personal preference in which case I'd really like to see custom themes or something like that. Or maybe I'm not the only one feeling like the UI lacks some intuitive structure which I believe is mostly caused by too much whitespace.

If you compare it to slack for instance, vector shows less content while feeling more cluttered IMO.

**Comment by richvdh** [comment]
> Why power users? Is desktop usage now synonymous with power user? :)

We probably want to keep the clean view for new users, so a compact view would be optional.

Perhaps "experienced users" is a better term than "power users".

**Comment by juliusv** [comment]
When I think about "compact", I'm thinking about something that looks more like Slack - that's not for power users either.

But sure, an even more compact view (like e.g. irssi / weecht) would be interesting as well :)

**Comment by discordianfish** [comment]
Let me first say that the riot redesign improved the situation a lot already. So great job on this!

I personally would like if we could:
- reduce the (vertical) whitespaces a bit more
- increase font weight for text messages
- align top of avatar with top of nickname (and maybe think about using (rounded) squares like Slack)

Which, to be frank, makes it simply a bit more slack looking.. But I think they nailed the design.

**Comment by juliusv** [comment]
@discordianfish for parts of that, https://github.com/vector-im/vector-web/issues/2300 could be relevant

**Comment by Dasoren** [comment]
I would like to cast my vote in favor of this.

I would really like it if riot-web, had a lot less vertical white space.

**Comment by allanday** [comment]
I'm pretty sure that the current layout can be incrementally improved in order to make it a bit more space efficient and easy to read, while still keeping the existing look and feel. Here's a screenshot, which I've modified (original on the left, modified version on the right):

![riot-chat-spacing](https://cloud.githubusercontent.com/assets/213068/23667815/07d35eba-0357-11e7-93de-41d32fce5696.png)

Aligning avatar images with the top of names makes conversations easier to read in my opinion.

**Comment by discordianfish** [comment]
@allanday That looks great!

**Comment by juliusv** [comment]
I'm going even more compact (and higher contrast) with a local style:

![design](https://cloud.githubusercontent.com/assets/538008/23678364/233f5924-0384-11e7-839d-a8df3c2449a9.png)

via this CSS:

```css
body {
    color: #2c2d30;
}

.mx_EventTile {
    padding-top: 0;
}

.mx_EventTile .mx_SenderProfile {
    color: #2c2d30 !important;
    font-weight: bold;
    opacity: 1;
}

.mx_RoomTile_name {
    color: #2c2d30;
}

.mx_EventTile_line {
    padding-top: 1px;
    padding-bottom: 1px;
}

.mx_EventTile_avatar {
    top: 0;
}
```

**Comment by tessgadwa** [comment]
I like this. Are you thinking of implementing this spacing for the default theme only, or also for the dark theme?

**Comment by juliusv** [comment]
@tessgadwa I'm not sure if you are replying to me or to @allanday, but so far I didn't have any plans, just using my local CSS overrides for myself. I'm happy for anyone to take it and turn them it into more proper Riot themes though.

**Comment by tessgadwa** [comment]
@juliusv I maintain a general Riot UX thread, for which this issue is certainly relevant. Helpful to have the CSS especially, thanks for including.

**Comment by grahamperrin** [comment]
Before I was aware of this issue, [2017-04-23](https://matrix.to/#/!DgvjtOljKujDBrxyHk:matrix.org/$14929204401482269lYJXr:matrix.org): 

> Familiarising myself with Riot, switching away from HexChat, I'm extremely pleased with most features.
> Early impressions, after a few hours' use with a relatively busy channel: [speed reading](https://en.wikipedia.org/wiki/Speed_reading) should be made easier.

[Later](https://matrix.to/#/!DgvjtOljKujDBrxyHk:matrix.org/$14929517101747633BnDFr:matrix.org): 

> In this context, I find avatars distracting and wasteful of space:

![2017-04-22 07-58](https://cloud.githubusercontent.com/assets/192271/25553484/15cbfe72-2cac-11e7-8710-57a6436a37fe.png)

> I guess that it's possible to make those disappear with e.g. Stylish.
> In that column: in lieu of each avatar I'd like to see just the name. Discreetly. A more effective use of space (mocked up a few hours ago):

![2017-04-22 07-58 mocked up](https://cloud.githubusercontent.com/assets/192271/25553487/33366d1c-2cac-11e7-97b5-bb59d4fbc368.png)

**Comment by uhoreg** [comment]
This is fixed in 0.10
